src/lib/models/deployment.js

```diff
@@ -52,12 +52,22 @@ module.exports = class Deployment {
     _.set(this.release, 'spec.template.spec.containers[0].env', envs)
   }
 
+  hostNetwork() {
+      return Boolean(_.get(this.spec, 'hostNetwork', false))
+  }
+
   ports() {
     return _.map(_.get(this.spec, 'ports', []), domain => {
-      return {
-        name: domain.name || 'defaultport',
-        containerPort: parseInt(domain.port, 10)
+      let ret = {
```

Comment:
This can be a `const`. Constant objects can have their properties mutated.

Reply:
:+1: